src/table/utils/handle-click.ts

```diff
@@ -37,7 +37,10 @@ export const handleClickToSort = (
   changeSortOrder: ChangeSortOrder,
   isInteractionEnabled: boolean
 ) => {
-  !(evt.target as HTMLElement).closest('#sn-table-head-menu-button') && isInteractionEnabled && changeSortOrder(column);
+  !document.querySelector('.sn-table-head-menu') &&
```

Comment:
I see. It is because the header doesn't update when you open and close the menu. This is a good thing. But that means we can't set a variable in the head component, since it will be outdated when we open and close the menu. And we can't use document.

Could you scope this part of the PR out? four verifails in one PR is already a lot

Reply:
The thing is that I am not sure we should solve it. The solutions that would work are pretty ugly and it now that I think about it, it is breaking the patterns we have for other context menus. Clicking outside on a button will both close the menu and activate the button